task3/src/config.js

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const databaseConfig = {
```

Comment:
No need to group all the variables from env file to js file & reexport them. Just be sure to call dotenv.config() as early as possible in your application & then use process.env in separate files

Reply:
I took this approach from https://softwareontheroad.com/ideal-nodejs-project-structure/ ( h1 Configurations and secrets ) which is stated in our task https://epa.ms/nodejs19-hw3 as best practice example. Unfortunately, sequelize api doesn't take an object as configuration and I have to pass those params as function arguments. 

So, I guess your proposal is better, I'm gonna move dotenv.config() call to app.js, and use variables as process.env.VAR_NAME

Reply:
I'd say that this particular part with grouping & reexporting is arguable, because though you'll avoid writing process.env.bla-bla-bla, you'll have to:
1) import you combined file all the time
2) every time remember that once you create a new env variable, you'll have to reexport it through this file
3) teach everybody in the team to do point 2 all the time & check if nobody uses directly process.env.bla-bla-bla, because it's totally possible or create some custom linter rule to avoid the practice

Reply:
👍

Reply:
Good point